purge-cluster.yml

```diff
@@ -223,6 +223,16 @@
     register: check_for_running_ceph
     failed_when: check_for_running_ceph.rc == 0
 
+  - name: see if ceph-disk-created data partitions are present
+    shell: "ls /dev/disk/by-partlabel | grep -q 'ceph data'"
+    failed_when: false
+    register: ceph_data_partlabels
+
+  - name: see if ceph-disk-created journal partitions are present
+    shell: "ls /dev/disk/by-partlabel | grep -q 'ceph journal'"
```

Comment:
On two setups (ubuntu and centos) it is named `ceph\x20journal` so this will give:

`ls /dev/disk/by-partlabel/ | egrep 'ceph\\x20journal'`.